app/helpers/digit-grouping.js

```diff
@@ -4,4 +4,4 @@ export function digitGrouping(value) {
   return parseFloat(value).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
 
-export default Ember.HTMLBars.makeBoundHelper(digitGrouping);
+export default Ember.Helper.helper(digitGrouping);
```

Comment:
@Robdel12 yeah it's no longer used. Wanna also remove that helper and it's test?